Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CreateProcessForm.java

```diff
@@ -408,8 +409,12 @@ private void processProcessHierarchy() throws ProcessGenerationException {
                 }
                 processDetails = metadata.getRows();
                 try {
+                    StructuralElementViewInterface docTypeView = rulesetManagementInterface
+                            .getStructuralElementView(docType, acquisitionStage, priorityList);
+                    String processTitle = docTypeView.getProcessTitle().orElseThrow(
+                        () -> new ProcessGenerationException(docType + " does not declare a process title."));
```

Comment:
I would prefere an empty process title instead of throwing an exception

Reply:
I cannot imagine that this can be wanted, but if you want it that way...